backends/platform/3ds/osystem-graphics.cpp

```diff
@@ -239,12 +388,12 @@ void OSystem_3DS::grabPalette(byte *colors, uint start, uint num) const {
 }
 
 void OSystem_3DS::copyRectToScreen(const void *buf, int pitch, int x,
-                                           int y, int w, int h) {
+                                   int y, int w, int h) {
```

Comment:
So, now you've done the rest of the changes, the pixel format used by the screen texture and the game are often the same. Wouldn't it be worth it to create a fast path for that case here, bypassing the whole color conversion business?